File: Wirtualny świat/java/src/organizm/trawa.java

package organizm;
import java.util.Random;
import swiat.swiat;


public class trawa extends roslina {
    public trawa(swiat WielkiSwiat, int x, int y)
    {
        super(WielkiSwiat, x, y);
        this.nazwa = 'T';
    }
    
    public void nowy(swiat WielkiSwiat, int x, int y)
    {
        organizm o = new trawa(WielkiSwiat, x, y);
    }
    public String nazwa()
    {
        return "Trawa";
    }

}

File: Wirtualny świat/C++/ConsoleApplication18/trawa.cpp
#include "stdafx.h"
#include "trawa.h"


trawa::trawa(swiat& ref, int x, int y) :roslina(ref, x, y)
{
	this->sila = 0;
	this->inicjatywa = 0;
	this->nazwa = 'T';
	this->specjalna_kolizja = 0;
}

char trawa::rysowanie() {
	return this->nazwa;
}
void trawa::nowe(swiat& ref, int x, int y)
{
	organizm *o = new trawa(ref, x, y);
	cout << "dodano: " << this->rysowanie() << " na polu: " << x + 1 << ':' << y + 1 << ":\n";	
}
trawa::~trawa()
{
}
